### ISSUE TITLE
Feature Request: Pencil Mode

### ISSUE THREAD

**Issue by nik3daz** [mmmewk/crosswordle#9]
So strategically, it's super useful to think about potential grid guesses and then rule them out with the wordle.

It would help a lot to have the pencil mode from the NYT crossword for the grid (even though I think it's totally useless for the NYTXW).

**Comment by mmmewk** [comment]
Implemented!

**Comment by nik3daz** [comment]
Wowowow!

#10xer #moonshots

**Comment by mmmewk** [comment]
@nik3daz Can you think of any way UI wise to make it more clear:
1) that pencil mode is on?
2) what pencil mode does?

**Comment by nik3daz** [comment]
Hmm. Personally, I feel that the big dialog on pencil mode enable/disable is pretty clunky! I think the highlight of the icon is an adequate indicator.

For new players, I feel many would read the instructions, so it probably makes sense to just leave bit of a note in there. e.g Pencil mode can be used to indicate what you think a square might be without actually using a guess.

For current players, maybe having a 'New!' tooltip for the fresh parts of the UI that they might want to experiment with would help? I think existing players will figure out what the tool does (and why it's useful), but might not notice it.

**Comment by mmmewk** [comment]
Hmm yeah the problem that I have been running into is that people don't ever click the ? icon so they don't read the How to Play part and rather just get confused. Yeah advanced keyboard at the bottom was just a way to bring it to attention of current users that its an option.

Probably can update it to be inside the settings menu pretty soon.

**Comment by nik3daz** [comment]
Maybe a "First Time?" tooltip under the ? icon? It could show after a few seconds on the player's first visit to the site?

**Comment by mmmewk** [comment]
Hmm might just open up the `How to Play` modal by default if they are a new user. Might be annoying for existing users who are using a different device, but could solve issues with people being confused on how to play. A lot of people play on mobile so hover based tooltips don't really work for that. But could have a tooltip show up immediately and then vanish after a few seconds

**Comment by nik3daz** [comment]
Yeah, the autodisplay and hide was what I was thinking, but showing the how
to play works too!
